specification/postgresqlhsc/resource-manager/readme.go.md

```diff
@@ -22,5 +22,5 @@ These settings apply only when `--tag=package-2020-10-05-privatepreview --go` is
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'package-2020-10-05-privatepreview' && $(go)
-output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-10-05-privatepreview/$(namespace)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-10-05-preview/$(namespace)
```

Comment:
why is this changed? there is no api version with this name...

Reply:
Why did we modify serverGroupsv2 apis? Are we going to introduce private dns for HSC?

Reply:
I don't know specifically what is causing it but this change is making WebScout scans to fail for 2020-10-05 private preview api version